.github/workflows/main.yml

```diff
@@ -186,6 +186,22 @@ jobs:
     - run: rustup target add aarch64-pc-windows-msvc
     - run: cargo check -p wasmtime --target aarch64-pc-windows-msvc
 
+  # Check whether `wasmtime` cross-compiles to aarch64-linux-android
+  checks_android:
+    name: Check Android AArch64
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        submodules: true
+    - uses: ./.github/actions/install-rust
+    - run: rustup target add aarch64-linux-android
+    - name: Setup Android SDK
+      uses: android-actions/setup-android@v2
+    - name: Install cargo-ndk
+      run: cargo install cargo-ndk
```

Comment:
Could this use a cached install like `mdbook` and I think `cargo-deny` to speed this up a little as well?

Additionally, like the FreeBSD support, could this get lumped into the generic "Check" job since it's quite quick?